tests/test_core/test_sequence.py

```diff
@@ -2732,18 +2732,17 @@ def test_empty_seqview_translate_position():
 @pytest.mark.parametrize("stop", (None, 10, 8, 1, 0, -1, -11))
 @pytest.mark.parametrize("step", (None, 1, 2, -1, -2))
 @pytest.mark.parametrize("length", (1, 8, 999))
-def test_seqview_seqlen_init(start, stop, step, length):
-    """seq_len is length of seq when None"""
-    seq_data = "0123456789"
+def test_seqview_seq_len_init(start, stop, step, length):
+    # seq_len is length of seq when None
+    seq_data = "A" * length
     sv = SeqView(seq_data, start=start, stop=stop, step=step)
     expect = len(seq_data)
     # Check property and slot
     assert sv.seq_len == expect
     assert sv._seq_len == expect
 
-    # Expect input seq_len to be correct if provided
-    got = SeqView(seq="ACTG", seq_len=length).seq_len
-    assert got == length
+    with pytest.raises(AssertionError):
```

Comment:
Make this a new test (make sure test function name is unique) and give it 3 different conditions using `pytest.mark.parametrize` (look at examples in the module for using this).
We want to check that the following conditions all fail
- `seq_len=0, seq="A"`
- `seq_len=01, seq=""`
- `seq_len=2, seq="A"`